### PR TITLE
OSDOCS-8964: Adding bug fix and blueprint notes in the microshift rel…

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.15

Issue:
[OSDOCS-8964](https://issues.redhat.com/browse/OSDOCS-8964)

Link to docs preview:
https://69950--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-15-release-notes#microshift-4-15-installation
https://69950--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-15-release-notes#microshift-4-15-installation-bug-fixes

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 1]
```diff
@@ -148,14 +148,13 @@ In the following tables, features are marked with the following statuses:
...
+[id="microshift-4-15-bug-fixes"]
+== Bug fixes
 
 //[discrete]
-//[id="microshift-4-15-installation-bug-fixes"]
-//==== Installation
-
+[id="microshift-4-15-installation-bug-fixes"]
+=== Installation
+* Previously, the `microshift-release-info` RPM depended on the `microshift` RPM. With this release, you no longer have any dependencies and {microshift-short} RPMs can be downloaded and installed independently.(link:https://issues.redhat.com/browse/OCPBUGS-22854[*OCPBUGS-22854*])
```
Comment (xenolinux) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:157]:
```suggestion
* Previously, the `microshift-release-info` RPM would depended on the `microshift` RPM. With this release, you no longer have any dependencies and you can download and install {microshift-short} RPMs independently.(link:https://issues.redhat.com/browse/OCPBUGS-22854[*OCPBUGS-22854*])
```

**Review by xenolinux — COMMENTED** [review]
I left a suggestion, otherwise looks good!

File: microshift_release_notes/microshift-4-15-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:157]: @ggiguash Please confirm/deny. The bug text doc changes say "the former," meaning the release info RPM ("This allows the former to be downloaded and installed independently when building MicroShift ISO images").

[the remark this answers is not in the document]
Comment (ggiguash) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:157]: Yes, your suggested text is correct.

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 2]
```diff
@@ -63,12 +63,12 @@ The following list provides update details:
...
-//=== Installation
+[id="microshift-4-15-installation"]
+=== Installation
 
-//[id="microshift-4-15-new-feature-for-use-at-installation"]
-//==== New feature for use during installation here
-//can include a note about a change in base RHEL image
+[id="microshift-4-15-blueprint-included"]
+==== Sample blueprints now included
+The `microshift-release-info` RPM now contains sample blueprints that can be used for image building. The blueprints include both {microshift-short} RPM packages and container image references.
```
Comment (dhensel-rh) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:71]: Do we need to show where the blueprint files are located, or is it up to the user to run the `rpm -ql <package>` to find them ?

Comment (ggiguash) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:71]: For a release note, we may want to skip the location to keep the text concise? 
In the docs, it should definitely be mentioned.

Comment (dhensel-rh) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:71]: @ggiguash That seems reasonable.

Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:71]: No location in the release notes. We can xref back to the section in the install docs once we have that done. Remember that this part of the doc is headlines and highlights.

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 3]
```diff
@@ -148,14 +148,13 @@ In the following tables, features are marked with the following statuses:
...
+[id="microshift-4-15-bug-fixes"]
+== Bug fixes
 
 //[discrete]
-//[id="microshift-4-15-installation-bug-fixes"]
-//==== Installation
-
+[id="microshift-4-15-installation-bug-fixes"]
+=== Installation
+* Previously, the `microshift-release-info` RPM depended on the `microshift` RPM. With this release, you can download and install the `microshift-release-info` RPM independently for use in image building.(link:https://issues.redhat.com/browse/OCPBUGS-22854[*OCPBUGS-22854*])
```
Comment (bscott-rh) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:157]:
```suggestion
* Previously, the `microshift-release-info` RPM depended on the `microshift` RPM. With this release, you can download and install the `microshift-release-info` RPM independently for use in image building. (link:https://issues.redhat.com/browse/OCPBUGS-22854[*OCPBUGS-22854*])
```

**Review by bscott-rh — COMMENTED** [review]
One tiny nit, and then I am good to merge this.

**Pull request merged**